Zend/tests/ctor_promotion_defaults.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+Constructor promotion with default values
+--FILE--
+<?php
+
+class Point {
+    public function __construct(
+        public float $x = 0.0,
+        public float $y = 1.0,
+        public float $z = 2.0
```

Comment:
In the RFC code example the trailing comma is allowed, but not here.

Which is correct?

Reply:
This is the subject of different RFC

Reply:
Yes, Here is PHP Wiki: https://wiki.php.net/rfc/trailing_comma_in_parameter_list